src/Field/BasicField.php

```diff
@@ -115,6 +115,12 @@ public function fetchFieldKey($fieldName)
      */
     public function fetchFieldType($fieldKey)
     {
+        // Check if this a clone field. If it is, return the field type of its clone.
+        $fieldKeyIds = explode('_', $fieldKey);
+        if (count($fieldKeyIds) == 4) {
```

Comment:
Actually you might be right, i forgot you can name the field keys too. May need to come up with another way.